chat/message/src/Message/Message.tsx

```diff
@@ -148,3 +160,39 @@ export const Message = forwardRef(
 Message.displayName = 'Message';
 
 export default Message;
+
+function VerifiedAnswerBanner({
+  verifier,
+  verifiedAt,
+  learnMoreUrl,
+}: NonNullable<MessageProps['verified']>) {
+  const text = useMemo(() => {
+    const textParts = [`Answer Verified`];
```

Comment:
Just want to confirm this is the expected behavior: if someone passes an empty object as the `verified` prop to `Message`, then the banner will only say "Answer Verified"

Reply:
Should the word "verified" be lowercase?

Reply:
Title case - see https://github.com/mongodb/leafygreen-ui/pull/2283#discussion_r1548503069

Reply:
> if someone passes an empty object as the verified prop to Message, then the banner will only say "Answer Verified"

Yeah correct though we'll guard against that specific case in our implementation - we ensure every answer has a `verifiedAt` value during import and hard code `verifier: "MongoDB Staff"` for now.